### PR TITLE
Support start and due date in epic detail view

### PR DESCRIPTION
**Pull request by maximilianruesch**
Adds support and selection of the start and due date in the epic detail view:

image
image
image

* Depends on #80
* Closes https://projectcanvas.atlassian.net/browse/SCRUM-66

### CODE REVIEW

File: src/components/EpicDetailView/EpicDetailView.tsx [hunk 1]
```diff
@@ -38,10 +43,29 @@ export function EpicDetailView({
   created: string
   updated: string
   closeModal: () => void
+  startDate: Date
+  dueDate: Date
 }) {
   const queryClient = useQueryClient()
   const reloadEpics = () => queryClient.invalidateQueries({ queryKey: ["epics"] });
 
+  const useDateMutation = (property: string) => useMutation({
```
Comment (aymka) [inline src/components/EpicDetailView/EpicDetailView.tsx:52]: Is it intentional for the epic's start date to precede its creation date?

**Review by aymka — CHANGES_REQUESTED** [review]
only one issue(or question?) but nonetheless, works excelllllent!

File: src/components/EpicDetailView/EpicDetailView.tsx (hunk 1 above)
Comment (maximilianruesch) [reply src/components/EpicDetailView/EpicDetailView.tsx:52]: Yes, as you may want to retroactively add epics you have done in the past. Or maybe even just create an epic that already started and you currently work on.

File: electron/providers/jira-server-provider/JiraServerProvider.ts
Comment (ValsiRod) [inline electron/providers/jira-server-provider/JiraServerProvider.ts:1]: The date picker is there for the server, but it doesn't let me save the start or end date if I make a change.

**Comment by maximilianruesch** [comment]
We decided to not support start and due dates at this time.

**Pull request closed without merging**